packages/hawtio/src/plugins/shared/jolokia-service.test.ts

```diff
@@ -44,20 +47,37 @@ describe('JolokiaService', () => {
 
   test('getJolokia - default', async () => {
     jolokiaService.getJolokiaUrl = jest.fn(async () => '/test')
-    Jolokia.prototype.list = jest.fn((path?: string | string[] | ListRequestOptions, opts?: ListRequestOptions) => {
+    Jolokia.prototype.list = jest.fn((...params: (string[] | string | SimpleRequestOptions)[]) => {
+      let path
+      let opts: SimpleRequestOptions | undefined = undefined
+      if (params.length === 2 && !Array.isArray(params[1]) && typeof params[1] === 'object') {
+        path = params[0] as string | string[]
+        opts = params[1] as SimpleRequestOptions
+      } else if (params.length === 1) {
+        if (!Array.isArray(params[0]) && !Array.isArray(params[0]) && typeof params[0] === 'object') {
+          opts = params[0] as SimpleRequestOptions
+        } else {
+          path = params[0] as string | string[]
+        }
+      }
+
       if (typeof path !== 'string') throw new Error('String is expected for path')
       if (!opts) throw new Error('No options set')
-      if (!opts.error) throw new Error('No error option set')
+      if (!opts.error) {
+        throw new Error('No error option set')
+      }
 
-      opts.error({
-        status: -1,
-        timestamp: 123456789,
-        request: { type: 'list', path: path },
-        value: null,
-        error_type: 'non-exist',
-        error: 'ERROR HAS OCCURRED',
-        stacktrace: 'not available',
-      })
+      ;(opts.error! as ErrorCallback)(
```

Comment:
same here

Reply:
I'm getting:
```
TS2349: This expression is not callable.
  Not all constituents of type "ignore" | ErrorCallback are callable.
    Type "ignore" has no call signatures.
```
without casting

Reply:
this was also added here:
https://github.com/hawtio/hawtio-next/blob/9125bb287c0bbc4a8e54f2dd720bcd6750ef341b/packages/hawtio/src/ui/util/ExpandableText.tsx#L297-L298

Reply:
ok, I can check that later